staff/carlota-vicens/beers-app/scripts/web-components-1.0.0.js

```diff
@@ -0,0 +1,153 @@
+'use strict';
+
+/**
+ * Web Components lib
+ * 
+ * @author manuelbarzi
+ * @version 1.0.0
+ */
+
+/**
+ * Constructs a component
+ */
+function Component(tag) {
+    var element = document.createElement(tag || 'div');
+    this.element = element;
+
+    // element.style.width = '100px';
+    // element.style.height = '100px';
+    this.size('auto', 'auto');
+
+    // element.style.borderColor = 'black';
+    // element.style.borderStyle = 'solid';
+    // element.style.borderWidth = '1px';
+    element.style.backgroundColor = 'silver';
+
+}
+
+Component.prototype.backgroundColor = function (color) {
+    this.element.style.backgroundColor = color;
+};
+
+Component.prototype.size = function (width, height) {
+    this.element.style.width = width;
+    this.element.style.height = height;
+}
+
+/**
+ * Constructs a panel
```

Comment:
It's okey to comment the functions but maybe you could add more information 

 `/**
 * Constructs a panel
 * 
 * @param {string} title - The title of the panel
 */`